lib/flutter_typeahead.dart

```diff
@@ -1274,6 +1275,11 @@ class TextFieldConfiguration<T> {
   /// Same as [TextField.textAlign](https://docs.flutter.io/flutter/material/TextField/textAlign.html)
   final TextAlign textAlign;
 
+  /// Same as [TextField.textDirection]
```

Comment:
Add link like the other properties. In this case (https://docs.flutter.io/flutter/material/TextField/textDirection.html)